monkey/infection_monkey/network/info.py

```diff
@@ -44,11 +38,11 @@ def get_host_subnets():
         if 'broadcast' in network:
             network.pop('broadcast')
         for attr in network:
-            network[attr] = network[attr].encode('utf-8').strip()
+            network[attr] = network[attr]
```

Comment:
I think we can delete all of this now, seems like this loop isn't doing anything.